e2e/registerENSFlow.spec.js

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable no-undef */
+/* eslint-disable jest/expect-expect */
+import { exec } from 'child_process';
+import { Contract } from '@ethersproject/contracts';
+import * as Helpers from './helpers';
+import registratABI from '@rainbow-me/references/ens/ENSETHRegistrarController.json';
```

Comment:
nit typo: `registratABI` should be `registrarABI`

Reply:
i don't know how to write